flink-runtime/src/test/java/org/apache/flink/runtime/operators/AbstractOuterJoinTaskTest.java

```diff
@@ -379,16 +380,18 @@ public void run() {
 
         taskRunner.join(60000);
 
-        assertFalse("Task thread did not finish within 60 seconds", taskRunner.isAlive());
+        assertThat(taskRunner.isAlive())
+                .withFailMessage("Task thread did not finish within 60 seconds")
+                .isFalse();
 
         final Throwable taskError = error.get();
         if (taskError != null) {
             fail("Error in task while canceling:\n" + Throwables.getStackTraceAsString(taskError));
```

Comment:
How about assertThat(taskError).withFailMessage("xxx").isNull()?

This class has 3 similar code, LeftOuterJoinTaskTest has 2, and RightOuterJoinTaskTest has 2, please update them if make sense to you.